DESCRIPTION

```diff
@@ -1,19 +1,19 @@
-Package: rLakeAnalyzer
-Title: Lake Physics Tools
-Maintainer: Luke Winslow <lawinslow@gmail.com>
-Version: 1.8.8
-Author: Luke Winslow, Jordan Read, Richard Woolway, Jennifer Brentrup, Taylor
-    Leach, Jake Zwart
-Description: Standardized methods for calculating common important derived
-    physical features of lakes including water density based based on
-    temperature, thermal layers, thermocline depth, lake number, Wedderburn
-    number, Schmidt stability and others.
-Imports:
-    plyr
-Suggests:
-    testthat
-License: GPL (>= 2)
-Packaged: 2014-07-06 09:09:24 UTC; Luke
-Repository: https://github.com/GLEON/rLakeAnalyzer
-Date/Publication: 2015-04-02 12:00:00
-RoxygenNote: 5.0.1
+Package: rLakeAnalyzer
+Title: Lake Physics Tools
+Maintainer: Luke Winslow <lawinslow@gmail.com>
+Version: 1.8.8
```

Comment:
Could you increment the version? I'd say a minor version jump for the addition of a function. 1.9.0